<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Full Editable Tree Model</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeView" name="treeView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>128</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addRowButton">
       <property name="text">
        <string>Add Row</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeRowButton">
       <property name="text">
        <string>Remove Row</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addChildButton">
       <property name="text">
        <string>Add Child</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
